### PR TITLE
Added solutions to exercise 2

### PR DESCRIPTION
**Pull request by andersle**
This adds solutions to exercise set 2.

### CODE REVIEW

**Comment by andersle** [comment]
@sroet: This one is ready for a review. 

Binder at: https://mybinder.org/v2/gh/andersle/chemometrics/add_solutions_to_exercise_2

File: solutions/exercise-002/02_solutions.ipynb [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
...qs5yH+cCuaq6NU72RYQlOxqpQO0t7MkFD9Bl63vwy++YuqI8qTPmQyU7RtR4TVUHeGZZDDCBMVKSLV/Bv/ry9X6/YeyiU+ttmpdIGtN47cNY7fwYhtEA9jkQ+pxNl++eIqdyW41DTaVAeCQCcxHwioiUishWEdkmIkm1RDKMlOXwW8mTUi7Ofa3OoaawwxSJwLQDfEALoL372PaFDSMetOvHR9VHcXHu67SSXTUONYUC4fUKjLuL81PgZvd+JyCpfTKGkUrosFtolVHMhbmv7xlrKgXCIyn6/RDODOZo4C6gBPgHcJi3phlG+lJ7R+mfrY/mMn2Np0vH0rr1Pkm3ixSKSIp+H6Gqg0RkAYCqbhWRbI/tMoy0JViB8d/vPoMXW37CkjO+g8N+mlgDG0AkAlPhxsMogIjsAzTt8ELDSGKCJUXO3t2L+c37M2j+g7ya8VPufXdV0sbEBBKJk/dh4GWgvYjcDswA7vHUKsNIY0LtDj268zTYtZZPpv0jZMmHZCMSJ+8zOL2R/gJsBX6qqi94bZhhpCuhdoeW5x3NGi3g/OxXCOyFmMwxMZHMYAAygQqgvAGvMQwjCkIlRf7fqIN4YtdPGOD7mkFZX9U4nqwxMZHUg7kFeB4ng7oL8JxbN9cwDA8IlxT5WbNT2V7dnEvzptZ4TbLGxETi5L0AGKiqJQAicidOf6S7vTTMMNKZUAXGrxo1gJfePJmLc6bQJWMDa6s7JnVMTCTLnfXUFKIsd8wwjDgzbmABXUfehAIX5b4e2/YpHhDJDGYrsNSt2aLAScAcEbkfQFWv99A+wzBqMWr4YbDlbC5b/SaXXfFf8DVPtEkhiURgprk3P1+EeqJhGHFiwG9gxQvw1WQ45JJEWxOSSLoK/DMehhiG0QAKjoS2B8Hix5NaYCLZRRotInNEZGOsyjWIyCQR+UpEvhSRV9ySnP5jE0RkpYiscNulGIZRGxE49HJYPws2Lkq0NSGJxMn7EPArnP5EsSrX8C7QT1UPBb4GJkCdzo6jgUdEJDPkWQwjnel7AWTmwJeP1f/cBBGJD2YtsFBVY5Z/pKrvBDz8AjjLvb+nsyOwWkT8nR0/j9W1DSNlyG0Lfc6mYsl/OHnuGL4tIulykyKZwdwEvC4iN4rI1f5bDG24BHjLvV8ArAk4FrKzozVeMwz4pNkZ+Kp2MWj3O0mZmxSJwNwOVAH5
```
Comment (sroet) [inline solutions/exercise-002/02_solutions.ipynb]: This comment is not correct in the final version of the code:
```suggestion
    "axs = axs.flatten() # Flatten the 5x1 array to a list of 5\n",
```

Comment (sroet) [inline solutions/exercise-002/02_solutions.ipynb]: ax is/ axs is not needed in the final version of the code:
```suggestion
    "for degree, fit in zip(degrees, fits):\n",
```

**Comment by andersle** [comment]
@sroet: Think this one is ready again (did just apply your suggestions).

**Comment by sroet** [comment]
LGTM, going in

**Pull request merged**